webapp/client/src/pages/KrankheitsKostenInfoPage.test.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
```

Comment:
Also change "KrankheitskostenInfoPage" to "MedicalExpensesInfoPage" in the file name and also the texts of the tests?